.github/workflows/tests.yml

```diff
@@ -83,7 +70,7 @@ jobs:
 
       - name: Upload logs
         if: failure()
-        uses: actions/upload-artifact@v2
+        uses: actions/upload-artifact@master
```

Comment:
unless we are looking for something unreleased, using tagged version would be safe and easy to manage for the future.

Reply:
We're using `master` pretty consistently elsewhere so I think it's worth doing so here too.

Reply:
oh ok, sounds good!